util/user_comparator_wrapper.h

```diff
@@ -17,6 +17,8 @@ namespace ROCKSDB_NAMESPACE {
 // perf_context.user_key_comparison_count.
 class UserComparatorWrapper final : public Comparator {
  public:
+  UserComparatorWrapper() = default;
```

Comment:
Can we initialize `user_comparator_` to nullptr to trip segfault on misuse, or at least comment this as creating an uninitialized UserComparatorWrapper?

Reply:
Good idea, will do.